src/aliases/api/types/alias.go

```diff
@@ -34,16 +37,12 @@ func FormatEntityAliases(ents []aliasent.Alias) []Alias {
 	return als
 }
 
-type AliasValue string
-
-type AliasKey string
-
-type RegistryName string
-
+// AliasRequest creates or modifies an alias value.
 type AliasRequest struct {
-	Value AliasValue `json:"value"`
+	Value []string `json:"value"`
```

Comment:
forgot the rest of the annotations, check how it's done in the other domains but you need a validate and an example annotation.